Fix apidocs links (#5494)
umbraco/UmbracoDocs: 11/umbraco-cms/reference/angular/README.md

If you have any questions regarding the above, feel free to reach out at [docs@umbraco.com](mailto:docs@umbraco.com) or raise an issue on the [official UmbracoDocs GitHub Issue Tracker](https://github.com/umbraco/UmbracoDocs/issues).
{% endhint %}

> Generally, you can find information about these via the [Backoffice UI API documentation](https://apidocs.umbraco.com/v11/ui). While this part of the documentation is auto-generated from the Umbraco source code, you can also find below some more in-depth descriptions and examples for these directives and services.

Reviewer: to resolve the vale error of having less than 25 words in this sentence (copied it from the change for v12 to have the same content). Also, this will need to be changed in v10 docs as well. 
```suggestion
Generally, you can find information about these via the [Backoffice UI API documentation](https://apidocs.umbraco.com/v11/ui).  This part of the documentation is auto-generated from the Umbraco source code.

Below you can find more in-depth descriptions and examples of AngularJS directives and services.
```
